plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -6943,6 +6935,10 @@ protected void verifyTableNameLengthFailurePermissible(Throwable e)
     @Override
     protected Optional<SetColumnTypeSetup> filterSetColumnTypesDataProvider(SetColumnTypeSetup setup)
```

Comment:
Add a test similar to what has been done for Postgres https://github.com/trinodb/trino/blob/fa081358b45997bb4758e26d0c2a49f6be543aa8/plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java#L178-L190